### DISCUSSION TITLE
Scrypted and eufy cams - works?

### DISCUSSION THREAD

**Discussion by xerabo** [koush/scrypted#760] [Q&A] [answered]
Hi, 

I would like to get my 4 eufy 2k cams to work with scrypted. Since I couldn't find a plug-in for eufy: does the combination work? If yes, could you give me directions on how to archive that.

Thanks for your help.

chris

**Comment by Balooski** [comment]
Have you had any luck on this?

**Comment by xerabo** [comment]
Sadly, no.

**Comment by akifbayram** [comment, marked as answer]
My Eufy E24 and C24 indoor cameras work fine with Scrypted/NVR/HomeKit using RTSP. Motion events are obviously not available.

**Reply by Vannixxo** [reply]
Why isn’t motion detection available? HKSV process motion by HomePod/ATV

**Reply by akifbayram** [reply]
Eufy doesn’t enable their cameras to share motion data (RTSP stream only) and HKSV requires the camera to report any motion detected. Try enabling the “OpenCV Motion Detection” plugin on the camera to detect motion and pass it through to HKSV (reload Homekit plugin after enabling).
